Unix/doc/omi/omi.texi

```diff
@@ -834,14 +834,14 @@ from the @samp{root/omi} namespace.
 
 The expression in braces represents the @b{instance name} for the given
 instance. This instance name has a single key, although an instance name may
-have multiple keys. Consider, for example, the following class.
+have multiple keys. Consider, for example, the following class with two keys.
 
 @example
 @verbatim
 class MyClass
 {
     [Key] String  Key1;
-    [Key] Uint32  Key2;
+          Uint32  Key2;

```

Comment:
A property name of Key implies Key. You may want to pick the third parameter and change the name